pkg/api/errors/errors.go

```diff
@@ -38,7 +40,7 @@ func NotFound(name string, componentType string, metadata map[string]string, grp
 		message,
 		legacyTag,
 	).
-		WithErrorInfo(reason, metadata).
+		WithErrorInfo(messages.RecordCompAndGet(reason, errorcodes.CategoryCommon), metadata).
```

Comment:
I think it actually is because it ends up at `UniversalHTTPHandler()` like [this comment as well](https://github.com/dapr/dapr/pull/8256#discussion_r1842830509), I'll remove.